uda_udf_test.py

```diff
@@ -0,0 +1,111 @@
+        self.log.info("Test completed")
+
+    def run_stress_threads(self) -> Tuple[CassandraStressThread, CassandraStressThread]:
+        self.log.info("Running mixed workload c-s thread alongside uda/udf stress thread...")
+        stress_cmd = self.params.get('stress_cmd')[0]
+        uda_udf_cmd = self.params.get('stress_cmd')[1]
+        stress_thread = self.run_stress_thread(stress_cmd=stress_cmd,
+                                               stats_aggregate_cmds=False,
+                                               round_robin=False)
+        uda_udf_thread = self.run_stress_thread(stress_cmd=uda_udf_cmd,
+                                                stats_aggregate_cmds=False,
+                                                round_robin=False)
+        self.log.info("Stress threads started.")
+        return stress_thread, uda_udf_thread
+
+    def prewrite_db_with_data(self) -> None:
+        self.log.info("Prewriting database...")
+        stress_cmd = self.params.get('prepare_write_cmd')
+        pre_thread = self.run_stress_thread(stress_cmd=stress_cmd, stats_aggregate_cmds=False, round_robin=False)
+        self.verify_stress_thread(cs_thread_pool=pre_thread)
+        self.log.info("Database pre write completed")
+
+    def _verify_udf_functions(self):
+        row_query = UDFVerification(name="row_query",
+                                    query=f"SELECT * FROM {self.KEYSPACE_NAME}.{self.CF_NAME} LIMIT 1",
+                                    verifier_func=lambda c2, c3, c7: all([c2, c3, c7]))
+        verifications = [
+            UDFVerification(name="lua_var_length_counter",
+                            query=f"SELECT {self.KEYSPACE_NAME}.lua_var_length_counter(c7) AS result "
+                                  f"FROM {self.KEYSPACE_NAME}.{self.CF_NAME} LIMIT 1",
+                            verifier_func=lambda c2, c3, c7, query_response: len(c7) == query_response.result),
+            UDFVerification(name="xwasm_plus",
+                            query=f"SELECT {self.KEYSPACE_NAME}.xwasm_plus(c2, c3) AS result "
+                                  f"FROM {self.KEYSPACE_NAME}.{self.CF_NAME} LIMIT 1",
+                            verifier_func=lambda c2, c3, c7, query_response: c2 + c3 == query_response.result),
+            UDFVerification(name="xwasm_div",
+                            query=f"SELECT {self.KEYSPACE_NAME}.xwasm_div(c2, c3) AS result "
+                                  f"FROM {self.KEYSPACE_NAME}.{self.CF_NAME} LIMIT 1",
+                            verifier_func=lambda c2, c3, c7, query_response: c2 // c3 == query_response.result),
+            # UDFVerification(name="xwasm_fib",
+            #                 query=f"SELECT {self.KEYSPACE_NAME}.xwasm_fib(c2) AS result "
+            #                       f"FROM {self.KEYSPACE_NAME}.{self.CF_NAME} LIMIT 1",
+            #                 verifier_func=lambda c2, c3, c7, query_response: self._fib(c2) == query_response.result)
```

Comment:
I decided to leave the fibonacci function out of the test, since it can be pretty resource consuming and I'm not using it in the aggregate either way.